src/components/Wizard/CreateVmWizard/initialState/vmSettingsTabInitialState.js

```diff
@@ -49,7 +49,7 @@ export const getVmSettingsInitialState = props => ({
 });
 
 export const getInitialVmSettings = (props = { createTemplate: false }) => {
```

Comment:
please add the `isV2vVmwareCrd` with `false` as a default. This could break things in the future if somebody for example chose to use it for `asHidden` which take distinction of `undefined` and `false` into account.

Reply:
ok, but the default `false` is recently handled via `CreateVmWizard` default prop.

Reply:
ok, just to be sure to have a better contract